cmd/minishift/cmd/root.go

```diff
@@ -83,25 +90,31 @@ var RootCmd = &cobra.Command{
 			isAddonInstallRequired bool
 		)
 
-		if !filehelper.Exists(constants.MakeMiniPath("addons")) {
+		constants.MachineName = constants.ProfileName
+		constants.Minipath = constants.GetMinishiftHomeDir()
+		constants.KubeConfigPath = filepath.Join(constants.Minipath, "machines", constants.MachineName+"_kubeconfig")
+
+		minishiftDirs.Home = constants.Minipath
```

Comment:
Moving forward once can then attach actual functions to the MinishiftDirs struct.